tutorials/groups.qmd

```diff
@@ -0,0 +1,242 @@
+---
+title: work with groups
+toc: true
+---
+
+```{julia}
+#| echo: false
+#| code-fold: true
+#| output: false
+using Pkg;
+cd(@__DIR__)
+Pkg.activate("."); # for reproducibility use the local tutorial environment.
+Pkg.develop(path="../") # a trick to work on the local dev version
+using Markdown
+```
+
+This is a short overview of group support in [`Manifolds.jl`](https://juliamanifolds.github.io/Manifolds.jl/) and how to get started working with them.
+
+Groups currently available in `Manifolds.jl` are listed in ``[group section](@ref GroupManifoldSection)``{=commonmark}.
+
+You can read more about the theory of Lie groups for example in [Chirikjian:2012](@cite).
+An example application of Lie groups in robotics is described in [SolaDerayAtchuthan:2021](@cite).
+
+First, let's load libraries we will use.
+`RecursiveArrayTools` is necessary because `ArrayPartition` defined in it is used as one of the possible representations of elements of product and semidirect product groups.
```

Comment:
We could for both packages maybe write `.jl` behind and link to the packages docs or repository?
I would also leave out the “defined in it”; one could just write “its” upfront ArrayPartition. We could even add a link to that type but maybe not so important.